tests/garage/experiment/test_snapshotter.py

```diff
@@ -47,6 +48,17 @@ def test_snapshotter(self, mode, files):
 
     def test_invalid_snapshot_mode(self):
         with pytest.raises(ValueError):
-            snapshotter = Snapshotter(
-                snapshot_dir=self.temp_dir.name, snapshot_mode='invalid')
+            snapshotter = Snapshotter(snapshot_dir=self.temp_dir.name,
+                                      snapshot_mode='invalid')
             snapshotter.save_itr_params(2, {'testparam': 'invalid'})
+
+    def test_conflicting_params(self):
```

Comment:
thanks for adding this!

Reply:
perhaps it belongs in a separate pr, although its a small change.

make sure to add it to your commit message.

Reply:
No prob, added to the commit msg.